internal/zero/cmd/env.go

```diff
@@ -10,6 +12,12 @@ const (
 	// PomeriumZeroTokenEnv is the environment variable name for the API token.
 	//nolint: gosec
 	PomeriumZeroTokenEnv = "POMERIUM_ZERO_TOKEN"
+
+	// BootstrapConfigFileName can be set to override the default location of the bootstrap config file.
+	BootstrapConfigFileName = "BOOTSTRAP_CONFIG_FILE"
+	// BootstrapConfigWritebackURI controls how changes to the bootstrap config are persisted.
+	// See controller.WithBootstrapConfigWritebackURI for details.
+	BootstrapConfigWritebackURI = "BOOTSTRAP_CONFIG_WRITEBACK_URI"
```

Comment:
I wonder if we should start calling such env variables that are required purely for deployment purposes something like `INTERNAL_` hinting they are not part of the public API / config surface and may be changed in the future.

Reply:
Good idea. For context, this was originally a command line flag but that was clunky to wire up all the way to here, but let me know if you think there is a better way to configure these. Adding an internal prefix seems good to me otherwise.

Reply:
On second thought, taking a look at some of the other internal variables we use ("CLUSTER_API_ENDPOINT", "CONNECT_SERVER_ENDPOINT" etc) vs other ones that could reasonably be user-serviceable ("POMERIUM_OTEL_ENDPOINT") the lack of prefix seems to be a consistent way to denote internal variables. What do you think?

Reply:
I see. yes those are confusing as well. the only reason to call `POMERIUM_OTEL_ENDPOINT` with `POMERIUM_` prefix was to avoid a potential conflict with OTEL SDK env vars.